.ci/jenkins/jenkins-trigger.groovy

```diff
@@ -24,7 +24,7 @@
 // Parameters which ngraph-unittest uses:
 String  PR_URL = CHANGE_URL
 String  PR_COMMIT_AUTHOR = CHANGE_AUTHOR
-String  JENKINS_BRANCH = "ngraph-core-r0.22.0.rc1"
+String  JENKINS_BRANCH = "aslepko/r0.22"
```

Comment:
seems unrelated

Reply:
@aslepko recommended this change to fix the CI error 
`our old docker hub is no longer online so the openmpi test there fails`